closure/protobuf/closure_proto_library.bzl

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("@rules_proto//proto:defs.bzl", "ProtoInfo")
```

Comment:
pls keep loads together. and for others as well

Reply:
Done.